Feature/BN-3101 Docs Update - Publish App Version Upload with different bundleId (#478)
appcircleio/appcircle-docusaurus: docs/publish-module/binary-management/index.md

When the upload is completed successfully, the relevant application version will appear in the list.

:::caution BUNDLE ID AND PACKAGE NAME MUST BE UNIQUE

> To maintain consistency, the uploading of iOS app versions with different Bundle IDs under the same Publish Profile is prohibited. All app versions are mandated to share the the same Bundle ID within the Publish Profile. 

Reviewer: ```suggestion
To maintain consistency, do not upload iOS app versions with different Bundle IDs under the same Publish Profile. All app versions must share the same Bundle ID within the Publish Profile.
```

> Similarly, for Android Publish Profiles, all app versions must exhibit identical Package Names within the Android Publish Profile.

Reviewer: ```suggestion
Similarly, for Android Publish Profiles, all app versions must have identical Package Names within the Android Publish Profile.
```

> The Bundle ID (iOS) and Package Name (Android) can be viewed beneath the Publish Profile name. Users can also verify this information by selecting the [Binary Information](https://docs.appcircle.io/publish-module/publish-information/binary-information) for an app version under actions menu.

Reviewer: ```suggestion
You can view the Bundle ID (iOS) and Package Name (Android) beneath the Publish Profile name. Users can also verify this information by selecting the [Binary Information](/publish-module/publish-information/binary-information) for an app version under the actions menu.
```
